es-rest-client-1.3/es-client-java-jersey/src/test/java/com/bazaarvoice/elasticsearch/client/GetTest.java

```diff
@@ -47,4 +48,14 @@
         assertTrue(get.getFields().isEmpty());
         assertEquals(get.getSource().get("field"), "value");
     }
+
+    @Test(expectedExceptions = IndexMissingException.class)
```

Comment:
Well i would not say you spend a lot of time I this case as expected exception is pretty straightforward. I do agree there is more control. I will update